kuzzleio/documentation-V2
# KZL-918 Add tags and sort search results

## What does this PR do?
Add tags and sort search results : 
* Index tags in Algolia
* Show tags in search results
* Search results are now sorted by tags

### How should this be manually tested?
* on a random golang sdk page : https://deploy-preview-175--kuzzle-doc-v2.netlify.com/sdk-reference/go/1/error-handling/ if you try to search for an sdk method you will see the pages for go first in the search results
* try using the search bar in another page on another language 

### Boyscout
move algolia file parser function to algolia helpers
